Samples/iOS-Swift/iOS-Swift/ViewControllers/ScopeViewController.swift

```diff
@@ -0,0 +1,61 @@
+import Sentry
+import UIKit
+
+class ScopeViewController: UIViewController {
```

Comment:
`m`: What drove you to add a view controller for this? Do we need UI to test this?

Reply:
> see it hurting us to keep it.

It's code. I only want to keep it if it makes sense. If you do, keep it. If not, ditch it. It seems like you're in favor of keeping it. No strong opinion from my side.